src/controllers/certificates/webhook_cert_controller.go

```diff
@@ -70,7 +73,13 @@ func (controller *WebhookCertificateController) Reconcile(ctx context.Context, r
 		return reconcile.Result{}, errors.WithStack(err)
 	}
 
-	certSecret := newCertificateSecret()
+	webhookDeployment := appsv1.Deployment{}
+	err = controller.apiReader.Get(controller.ctx, types.NamespacedName{Name: webhook.DeploymentName, Namespace: controller.namespace}, &webhookDeployment)
```

Comment:
so the technically the webhook certs are "owned" by the operator (because its the one creating it), no ?

Reply:
true